assets/css/query-builder.css

```diff
@@ -1,5 +1,43 @@
 /* @override http://localhost:4000/assets/css/query-builder.css */
 
+/* @override http://localhost:4000/assets/css/query-builder.css */
+
+@-webkit-keyframes rotating /* Safari and Chrome */ {
```

Comment:
Are you sure we're going to need any sort of loading indicator? I'm not. And if we do, we'd probably use whatever pubmed already has.